### DISCUSSION TITLE
Account banned on Vercel for free tier quota. Will I ever get it back?

### DISCUSSION THREAD

**Discussion by drew-harris** [vercel/community#622] [Help] [answered]
One of my websites recently experienced a surge of traffic that put me over the free tier on a hobby account. Instead of paying for the pro-tier, I would prefer to just take the website down. Will my account be back next month or do I need to do something special to unblock it?

**Comment by swarnava** [comment, marked as answer]
You need to contact our Support team from your registered email: https://vercel.com/support/request. Thanks!
